doc/install.sgm

```diff
@@ -1,123 +1,98 @@
 <chapter id="install">
  <title>Installation</title>
 
-        <sect1 id="install-download">
-            <title>
-                Download
-            </title>
-            <para>
-                <application>pgSphere</application> is not part of the <application>PostgreSQL</application> software.
-                You can download it from the <application>pgSphere</application> homepage
-                <ulink url="https://github.com/postgrespro/pgsphere"><citetitle>https://github.com/postgrespro/pgsphere</citetitle></ulink>
-            </para>
-        </sect1>
+ <sect1 id="install-download">
+  <title>Download</title>
 
-        <sect1 id="install-sect2">
-            <title>
-                Installation
-            </title>
-            <para>
-                You will need <application>PostgreSQL
-                9.1</application> or above. We assume that you have
-                <application>PostgreSQL</application> already compiled and
-                installed. Please note: Depending on your system configuration mostly you have to be logged in as the system
-                superuser.
-            </para>
-            <para>
-                There are two ways to compile <application>pgSphere</application>.
-                The first is to copy the sources into the contribution directory of
-                <application>PostgreSQL</application>'s source tree
-                (<filename>POSTGRESQL_SRC/src/contrib</filename>).
-                Then, change into <filename>POSTGRESQL_SRC/src/contrib</filename>.
-                If the sources are not yet installed and the directory
-                <filename>pg_sphere</filename> does not exist, take the
-                gzipped <application>pgSphere</application> sources ( e. g.,
-                <filename>pg_sphere_xxx.tgz</filename> ) and run:
-            </para>
-            <programlisting>
-<![CDATA[shell> tar -xzf path/to/pg_sphere_xxx.tgz]]>
-            </programlisting>
-            <para>
-                Now, change into the <filename>pg_sphere</filename>
-                directory and run :
-            </para>
-            <programlisting>
+  <para>
+   &pgsphere; is not the part of the &postgresql; software. You can download
+   the latest release sources from the
```

Comment:
Remove "sources" from this line. Saying "latest release" is sufficient and more succint.